scrapers/my-slavegirl.yml

```diff
@@ -0,0 +1,26 @@
+name: "My Slavegirl"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 'my-slavegirl.com/en/collections/'
```

Comment:
`'` are not needed here
just for cosmetic reasons and to maintain the same format as the  rest of the scrapers i would remove them
```yaml
      - my-slavegirl.com/en/collections/
```